sunshine/confighttp.cpp

```diff
@@ -221,6 +221,27 @@ void getTroubleshootingPage(resp_https_t response, req_https_t request) {
   response->write(header + content);
 }
 
+void getFaviconImage(resp_https_t response, req_https_t request) {
+  print_req(request);
+
+  std::string content = read_file(WEB_DIR "images/favicon.ico");
+  response->write(content);
+}
+
+void getSunshineLogoImage(resp_https_t response, req_https_t request) {
+  print_req(request);
+
+  std::string content = read_file(WEB_DIR "images/logo-sunshine-45.png");
+  response->write(content);
+}
+
+void getFontAwesomeCss(resp_https_t response, req_https_t request) {
+  print_req(request);
+
+  std::string content = read_file(WEB_DIR "third_party/font-awesome.5.15.4.all.min.css");
+  response->write(content);
+}
```

Comment:
Can we serve entire `fonts` directory (recursively)?

Reply:
Same thought than the other comment. Plus don't forget to add Content-Type:text/css HTTP header.